cerbero/packages/wix.py

```diff
@@ -111,8 +118,12 @@ def _format_path_id(self, path, replace_dots=False):
         return ret
 
     def _format_dir_id(self, string, path, replace_dots=False):
-        return self._format_id(string, replace_dots) + '_' +\
+        ret = self._format_id(string, replace_dots) + '_' +\
             self._format_path_id(path, replace_dots)
+        # Wix Id length is limited to 72 characters.
+        if len(ret) > 72:
```

Comment:
Isn't this the same trunking algorithm used in _format_path_id ?

Reply:
not exactly because _format_path_id() may add a number after, so I reserve 3 more characters.

Reply:
format_path_id adds a number to avoid an ID collision having 2 ids that are identical. When trunking the ID to X characters if the X characters are identical you will have 2 id's that are similar.
The logic that trunks the ID should be similar in all places, where you trunk the ID to have X character and use a number to avoid duplicate ID's.

Take this as example with a max length of 10:
12345678910ABC
12345678910XYZ

If you trunk this at 10 you end up with a collision:
12345678910
12345678910

The  algorithm should instead do:
1234567_00
1234567_01

Reply:
You're right, I'm gonna improve it in a fix OPE-677-Fixes branch.